### ISSUE TITLE
 Enhancements for Web Project

### ISSUE THREAD

**Issue by ChaithanyaKrishna28** [Yash-Parsana/CrazyCoderWebApp#9]
Hello team,

I'd like to propose some enhancements to our web project to improve user experience and visual appeal. These changes mainly focus on adding hover effects to various elements.

1. **Navbar Hover Effect:**
   Currently, our navbar lacks any hover effect, which could enhance user interaction and provide visual feedback. Adding a hover effect to the navbar items would make navigation more intuitive and engaging.

2. **Banner Hover Effect:**
   The banner section featuring upcoming events from platforms like Atcoder, Codechef, Codeforces, Hackerrank, Hackerearth, and Leetcode could benefit from a hover effect. Implementing a subtle hover effect on these elements would make the banner more attractive and encourage user engagement.

3. **Footer Enhancements:**
   Our footer is missing hover effects, particularly on the icons. Introducing a hover effect, such as scaling, to the footer icons would add a polished touch to our website and make it more visually appealing. Additionally, incorporating transition effects would provide a smooth visual experience when interacting with the footer icons.

**Proposed Solution:**
- Implement hover effects on navbar items.
- Add hover effects to the banner section for upcoming events.
- Enhance the footer by adding hover effects and transition effects to the icons.

**footer section issue**
![footer_icons_issue](https://github.com/Yash-Parsana/CrazyCoderWebApp/assets/122201792/c08db726-efaa-4ac8-b61a-84ee04007d1a)

**Proposed Solution for navbar**
![issue1_resolve](https://github.com/Yash-Parsana/CrazyCoderWebApp/assets/122201792/59053283-9f19-44c3-af25-32b8523cf096)

**Issue with the navbar**
![navbar_banner_issu](https://github.com/Yash-Parsana/CrazyCoderWebApp/assets/122201792/e26535cf-1087-4738-8a05-2c37f68000aa)
Let's discuss and prioritize these enhancements to improve our web project's user experience. Looking forward to your feedback!
Best regards,
Chaithanya Krishna

**Comment by bhavanishanker-png** [comment]
Please allocate this task to me. I'm capable of tackling it as I possess all the requisite skills to address the matter at hand.

**Comment by Yash-Parsana** [comment]
This issue assigned to you @ChaithanyaKrishna28 

Implement below hover effects.
For nav - it should be same as active option (blue colored text)
For banner - Little transperant bg color like (refer this site (buttons) : https://yash-parsana-portfolio.vercel.app/ )
For footer - you may implement of your choice but use simple effect.

**Comment by ChaithanyaKrishna28** [comment]
> This issue assigned to you @ChaithanyaKrishna28 
> 
> Implement below hover effects.
> For nav - it should be same as active option (blue colored text)
> For banner - Little transperant bg color like (refer this site (buttons) : https://yash-parsana-portfolio.vercel.app/ )
> For footer - you may implement of your choice but use simple effect.

I would love to work on this issue.
Can you assign me this under gssoc24

**Comment by Yash-Parsana** [comment]
It is already assigned to you.

**Comment by ChaithanyaKrishna28** [comment]
Can you please provide the installation guide in the readme.md.
Can you discuss how to deploy the project in locally as when i tried to run to npm run preview the project is not rendering on the web browsser.(Note:I have installed all the node modules also).
Can you(@Yash-Parsana ) please assist me in this issue.

**Comment by manpreet244** [comment]
I would like to work on this issue under GSSOC , I would add animation too on hover effects .

**Comment by Yash-Parsana** [comment]
@ChaithanyaKrishna28 npm run dev is command to run project. you can find it in package.json file.
please complete it by tomorrow.

**Comment by ChaithanyaKrishna28** [comment]
> @ChaithanyaKrishna28 npm run dev is command to run project. you can find it in package.json file.
> please complete it by tomorrow.

Yeah I tried to run the command but it's showing many warnings and errors.
I will check and complete it by tomorrow!.
Thank you for your assistance.

**Comment by Yash-Parsana** [comment]
can you share error screenshots?

**Comment by ChaithanyaKrishna28** [comment]
@Yash-Parsana have deployed it sucessfully and made the following chnages in the nav bar and Selection pannel.


https://github.com/Yash-Parsana/CrazyCoderWebApp/assets/122201792/6d8e338c-1d29-481c-8978-c423c0a4499b



I would like to work on the leader board and the footer.
If you need any modifications please mention so that i will go on with them

Thank You.

**Comment by Yash-Parsana** [comment]
> This issue assigned to you @ChaithanyaKrishna28
> 
> Implement below hover effects. 
> For nav - it should be same as active option (blue colored text) 
> For banner - Little transperant bg color like (refer this site (buttons) : https://yash-parsana-portfolio.vercel.app/ ) 
> For footer - you may implement of your choice but use simple effect.

@ChaithanyaKrishna28 please follow this instructions.
